bot/cogs/utilities.py

```diff
@@ -151,7 +153,13 @@ async def ping(self, ctx):
         """Sends the ping between the bot and the discord API."""
         await ctx.send(f"Ping is {round(self.bot.latency * 1000)}ms")
         return self.bot.latency
-
+    
+    @command(brief="Shows good quotes")
+    async def quotes(self, ctx):
+        """Sends a random quote from the list in the quotesList file"""
+        embed_quote = discord.Embed(title=random.choice(quotes), color=discord.Color.green())
```

Comment:
You should not be redefining `quotes` ... reusing variable names is guaranteed to bite you in the ass some day and cause some unpredictable behaviour, on top of making it confusing for the reader/maintainer.